src/Organizations/OrganizationsList/OrganizationsListContainer.js

```diff
@@ -60,15 +63,14 @@ const OrganizationsListContainer = ({ mutator, location, history }) => {
     refreshList,
   } = useList(false, loadOrganizations, loadOrganizationsCB, RESULT_COUNT_INCREMENT);
 
-  useLocationReset(history, location, '/organizations', refreshList);
```

Comment:
useLocationReset makes things bad in ui-orders, because there is used history.replace in save/load filters from localStorage